app/celery/tasks.py

```diff
@@ -95,9 +97,9 @@ def copy_redaction_failed_pdf(source_filename):
     obj.copy(copy_source, ExtraArgs={'ServerSideEncryption': 'AES256'})
 
 
-@notify_celery.task(name='create-pdf-for-templated-letter')
+@notify_celery.task(bind=True, name='create-pdf-for-templated-letter')
```

Comment:
I think the default number of retries and the default delay between retries are good, but it might be useful to add them in as arguments anyway (even if you keep the defaults). It would help to make it obvious what they were and mean that we don't get caught out if the defaults change in future.

Reply:
Sure, I've done that 👍.